hazelcast/src/main/java/com/hazelcast/instance/GroupProperty.java

```diff
@@ -293,6 +293,8 @@
     ENABLE_JMX_DETAILED("hazelcast.jmx.detailed", false),
     JMX_UPDATE_INTERVAL_SECONDS("hazelcast.jmx.update.interval.seconds", 5, SECONDS),
 
+    DISABLE_ITERATING_MAP_STATS("hazelcast.management.disable.iterating.map.stats", false),
```

Comment:
our all `enabling/disabling` property names end with `X.X.X.enabled`, why you don't use that formation?